freestyle-twitter-future/src/main/scala/future.scala

```diff
@@ -0,0 +1,37 @@
+package freestyle
+package twitter
+
+import com.twitter.util.Future
+import cats.MonadError
+
+class FreestyleTwitterMonad extends MonadError[Future, Throwable] with Capture[Future] {
+
```

Comment:
`catbird` (a project with cats type class instances for Twitter data types) does override `ap` as you did before ([catsbird instances future](https://github.com/travisbrown/catbird/blob/master/util/src/main/scala/io/catbird/util/future.scala)).